Source/BestProjectEver/Private/Character/BPE_PlayerCharacter.cpp

```diff
@@ -503,7 +504,7 @@ void ABPE_PlayerCharacter::GetLifetimeReplicatedProps(TArray<FLifetimeProperty>&
 	
 	DOREPLIFETIME_CONDITION(ABPE_PlayerCharacter, OverlappingWeapon, COND_OwnerOnly);
 	DOREPLIFETIME(ABPE_PlayerCharacter, CurrentWeapon);
-	DOREPLIFETIME(ABPE_PlayerCharacter, bIsAiming);
+	DOREPLIFETIME_CONDITION_NOTIFY(ABPE_PlayerCharacter, bIsAiming, COND_None, REPNOTIFY_Always);
```

Comment:
why do we need the REPNOTIFY_Always?

Reply:
Ou we don't, I did it before because it wasn't updated on the clients. Then I realized that it was because I changed bIsAiming on the client and then on the server so I assign it only if it has authority now, but I forgot to change that 
![image](https://user-images.githubusercontent.com/85327912/173094955-1ffc1f99-6670-4bec-9473-25bb08332537.png)